challenges/pt-br/front-challenge.md

```diff
@@ -1,32 +1,42 @@
 # Hash Teste Front-end
 
-Antes de começar, leia os nossos [key values](https://www.keyvalues.com/hash) para entender um pouco sobre o que nós priorizamos no desenvolvimento e faça o seu melhor, pois iremos avaliar o teste como se fosse seu melhor esforço ;)
+Antes de começar, leia os nossos [key values](https://www.keyvalues.com/hash) para entender um pouco sobre o que nós priorizamos no desenvolvimento e **faça o seu melhor, pois iremos avaliar o teste como se fosse seu melhor esforço** ;)
 
 O teste consiste desenvolver um simulador de antecipação de recebíveis composto por:
 
-- App com formulário para o usuário interagir.
-- Biblioteca para executar o cálculo de recebíveis.
-
-**Lembre-se, escreva testes automatizados.**
+> - App com formulário para o usuário interagir.
+> - Biblioteca para executar o cálculo de recebíveis
+>   - Parcelamento máximo de 12x
 
 Envie o resultado do seu desafio para dev@hash.com.br (ele pode ser open source!). Em até uma semana marcaremos uma conversa com você após analisarmos seu desafio.
 
 ## Restrições
 
 1.  Não é permitido utilizar frameworks e/ou bibliotecas de UI, como React, Vue.js, Angular e JSX (queremos ver a sua habilidade com JS e DOM).
 2.  São permitidas ferramentas modernas de desenvolvimento como TypeScript, Babel, eslint, webpack, assim como o uso de polyfills (e outras ferramentas para melhorar o suporte a browsers, como o Modernizr) e/ou bibliotecas para testes.
-3.  São permitidos pré-processadores de CSS ou ferramentas CSS-in-JS.
+3.  São permitidos pré-processadores de CSS e/ou ferramentas CSS-in-JS.
 4.  Não é uma regra, mas evite usar lodash, underscore, ramda e similares.
 
-## Avaliação
+## Como Avaliaremos seu Teste Técnico
+
+Sua performance será avaliada com base nos seguintes pontos:
```

Comment:
I think the way this is written is too specific and will probably bias the solution.

I like something along the lines of this (IMO well redacted job description): https://hashlab.slack.com/archives/CCLHHATHC/p1547147343000800

```
Evaluation Criteria
--------------
​
1. The problems are solved efficiently and effectively, the application works as expected.
2. The application is supplied with the setup scripts. Consider using docker and a one-liner setup.
3. You demonstrate the knowledge on how to test the critical parts of the application. We **do not require** 100% coverage.
4. The application is well and logically organised.
5. The submission is accompanied by documentation with the reasoning on the decisions taken.
6. The code is documented and is easy to follow.
7. The answers you provide during code review.
8. An informative, detailed description in the PR.
9. Following the industry standard style guide.
10. A git history (even if brief) with clear, concise commit messages.
```

I see a lot of common points with what you've written (like item 10), but they are more open.

Reply:
I like that. I will make the changes...